projects/js-packages/publicize-components/src/hooks/use-media-restrictions/restrictions.js

```diff
@@ -126,6 +132,12 @@ export const RESTRICTIONS = {
 		allowedMediaTypes: [ 'image/jpg', 'image/jpeg', MP4, MOV, VIDEOPRESS ],
 		image: {
 			maxSize: 8,
+			minWidth: 320,
+			maxWidth: 1440,
+			aspectRatio: {
+				min: 4 / 5,
+				max: 1.91 / 1,
```

Comment:
Do you think we should just keep this as 1.91?

```suggestion
				max: 1.91,
```